business_requirement/views/business_view.xml

```diff
@@ -0,0 +1,149 @@
+                        <button name="action_button_confirm" states="draft" string="Confirm" type="object" groups="business_requirement.group_business_requirement_user" class="oe_highlight"/>
+                        <button name="action_button_back_draft" string="Back To Draft" type="object" attrs="{'invisible': [('state', 'in', ('draft','done'))]}" groups="business_requirement.group_business_requirement_user" class="oe_highlight"/>
+                        <button name="action_button_approve" states="confirmed" string="Approve" type="object" groups="business_requirement.group_business_requirement_manager" class="oe_highlight"/>
+                        <button name="action_button_done" states="approved" string="Done" type="object" groups="business_requirement.group_business_requirement_manager" class="oe_highlight"/>
+                        <button name="action_button_cancel" string="Cancel" type="object" attrs="{'invisible': [('state', 'in', ('done','cancel'))]}" groups="business_requirement.group_business_requirement_user" class="oe_highlight"/>
+                        <field name="state" widget="statusbar" statusbar_visible="draft,confirmed,approved,done,cancel"/>
+                    </header>
+                    <sheet>
+                        <h1>
+                            <field name="name" class="oe_inline" readonly="1"/>
+                        </h1>
+                        <group>
+                            <group>
+                                 <field name="description"/>
+                             </group>
+                        </group>
+                        
+                        <notebook>
+                            <page string="Business Requirement">
+                                <group string="Customer Story">
+                                     <field name="business_requirement" widget="html" nolabel="1"/>
+                                </group>
+                                <group string="Scenario">
+                                    <field name="scenario" widget="html" nolabel="1"/>
+                                </group>
+                                <group string="Gap">
+                                     <field name="gap" widget="html" nolabel="1"/>
+                                </group>
+                            </page>
+                            <page string="Sub Business Requirement">
+                                <field name="business_requirement_ids" readonly="1" context="{'form_view_ref':'view_business_requirement_form', 'tree_view_ref':'view_business_requirement_tree'}"/>
+                            </page>
+                            <page string="Other Information">
+                                <group col="4">
+                                    <field name="categ_id"/>
+                                    <field name="parent_id"/>
+                                    <field name="level"/>
+                                    <field name="drop"/>
```

Comment:
As mentioned before, dropped should go away as it's the same as the cancelled state